packages/govuk-frontend/src/govuk/components/button/button.yaml

```
@@ -2,15 +2,15 @@ params:
   - name: element
     type: string
     required: false
-    description: Whether to use an `input`, `button` or `a` element to create the button. In most cases you will not need to set this as it will be configured automatically if you use `href` or `html`.
+    description: Whether to use an `input`, `button` or `a` element to create the button component. In most cases you will not need to set this as it will be configured automatically if you use `href` or `html`.
```

"Whether to use" indicates a boolean to me. Can this be something like

```suggestion
    description: Element to use to create the button component. It can be  `input`, `button` or `a`. In most cases you will not need to set this as it will be configured automatically if you use `href` or `html`.
```

---

Ah I tried to steer clear of changing what was already in there unless it was wrong, but good point

For consistency, shall we follow how the `type` choices are shown?

```suggestion
    description: HTML element for the button component – `input`, `button` or `a`. In most cases you will not need to set this as it will be configured automatically if you use `href` or `html`.
```